addons/huntir/stringtable.xml

```diff
@@ -273,5 +256,22 @@
             <Turkish>HuntIR Ekranı</Turkish>
             <Hungarian>HuntIR Monitor</Hungarian>
         </Key>
+        <Key ID="STR_ACE_HuntIR_ALT">
```

Comment:
not sure why this is moved down?
but don't worry it's not a problem, we can just re-sort it

Reply:
I accidentally uploaded STR_ACE_HuntIR_ALT replaced to weird letters, and I think there was a mistake in the process of fixing that.
I apologize.